sku/client.go

```diff
@@ -125,6 +125,10 @@ func (c Client) Update(id string, params *stripe.SKUParams) (*stripe.SKU, error)
 			body.Add("image", params.Image)
 		}
 
+		for k, v := range params.Attrs {
+			body.Add(fmt.Sprintf("attributes[%v]", k), v)
```

Comment:
Ah! Sorry this was my mistake. I was assuming that `Attrs` here was a slice for some reason, but now that I go and look at it, this is obviously not the case. Thanks for the clarification!